package.json

```diff
@@ -4,7 +4,9 @@
   "description": "",
   "main": ".eleventy.js",
   "scripts": {
-    "start": "npx eleventy --serve"
+    "serve:dev": "gulp build & ELEVENTY_ENV=dev npx @11ty/eleventy --serve",
+    "build:staging": "gulp build --production & ELEVENTY_ENV=staging npx @11ty/eleventy --serve",
```

Comment:
Everything looks good locally, for me! Thank you. 

However, this has been pushed to staging/dev and on Netlify (with the config you gave), and it keeps getting stuck at "building". I believe it’s because the `build:staging` command includes the `--serve` flag, which is for local development (it does `--watch`). I think the right way for staging would be `gulp build & ELEVENTY_ENV=dev npx @11ty/eleventy`. I will change it now.

Reply:
See the doc here: https://www.11ty.dev/docs/usage/

Reply:
Oh ok! I missed that -- thank you for catching it!

Reply:
@thinkbulecount2 ...and it worked! This was done for the two build commands staging and prod).

Also agreed with your renaming of things. I think we should rename dev.oa.works to staging.oa.works to make this clearer, since `serve:dev` is for local development if I understand correctly?

Reply:
Yes! I agree too!

Reply:
Nice one!! :) I’m merging it all to `main`. It looks great.

Reply:
Checks for what env:
```function isEnv (env) {
  if (env === "prod") {
    return process.env.BASE_URL = 'https://oa.works';
  } else if (env === "staging") {
    return process.env.BASE_URL = 'https://dev.oa.works';
  } else {
    return process.env.BASE_URL;
  }
}```

Reply:
Sure thing!